content-node/docker-compose/setup.sh

```diff
@@ -154,7 +169,11 @@ if [ -f ".env" ]; then
     if [[ ! $setup_again =~ ^[Yy]$ ]]; then
         log_info "Keeping existing .env file."
         echo
-        log_info "To start the content node, use: ${GREEN}docker compose up -d${NC}"
+        if [ "$AUTO_UPGRADE" = "true" ]; then
+            log_info "To start the content node with auto-upgrade, use: ${GREEN}docker compose --profile autoupdate up -d${NC}"
```

Comment:
will this also start the node?

Reply:
the script won't start the node and just prints the necessary command